SPECS/python-pyroute2/python-pyroute2.spec

```diff
@@ -1,90 +1,60 @@
 %global srcname pyroute2
 
-Summary:        Pure Python netlink library
-Name:           python-%{srcname}
-Version:        0.6.5
-Release:        3%{?dist}
-License:        GPLv2+
-URL:            https://github.com/svinota/%{srcname}
-Source0:        https://pypi.io/packages/source/p/pyroute2/%{srcname}-%{version}.tar.gz
-Source1:        %{srcname}.core-%{version}.tar.gz
-Source2:        %{srcname}.nslink-%{version}.tar.gz
-Source3:        %{srcname}.nftables-%{version}.tar.gz
-Source4:        %{srcname}.ethtool-%{version}.tar.gz
-Source5:        %{srcname}.ipset-%{version}.tar.gz
-Source6:        %{srcname}.ipdb-%{version}.tar.gz
-Source7:        %{srcname}.ndb-%{version}.tar.gz
-BuildArch:      noarch
-
-%description
-PyRoute2 provides several levels of API to work with Netlink
-protocols, such as Generic Netlink, RTNL, TaskStats, NFNetlink,
+%global _description \
+PyRoute2 provides several levels of API to work with Netlink\
+protocols, such as Generic Netlink, RTNL, TaskStats, NFNetlink,\
 IPQ.
 
-%package -n python3-%{srcname}
-Summary:        %{summary}
-BuildRequires:  python3-devel
-BuildRequires:  python3-setuptools
- 
-%description -n python3-%{srcname}
-PyRoute2 provides several levels of API to work with Netlink
-protocols, such as Generic Netlink, RTNL, TaskStats, NFNetlink,
-IPQ.
+Name: python-%{srcname}
+Version: 0.7.12
+Release: 1%{?dist}
+Vendor: Microsoft Corporation
+Distribution: Azure Linux
+Summary: Pure Python netlink library
+License: GPL-2.0-or-later OR Apache-2.0
```

Comment:
This reminded me we were trying to use SPDX short identifiers for licenses at some point. I've got no problem here but now I'm not sure, if this is written down anywhere, so it might be worth clarifying for any future plans we may have.

Reply:
Happy to chat more about this, but I'll assume this is fine as-is for the purposes of this PR.